index.ts

```diff
@@ -1,3 +1,4 @@
 export * from "./typechain";
 export * from "./src/DeploymentUtils";
 export * from "./utils";
+export * from "./src/svm";
```

Comment:
The program getters, types, and IDLs can be imported directly from `@across-protocol/contracts`, as well as utility functions (we should move the latter to the SDK next).